package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "js-organization",
+  "version": "1.0.0",
```

Comment:
Can we start with a lower version number here, so `0.1.0` or even `0.0.1` and keep the `1.0.0` for a first version where structured has been relatively settled and we have got something which we would publish officially?

Reply:
👍 dropped to 0.0.1 as we have nothing published yet